### PR TITLE
Update PHP version

### PR DESCRIPTION
**Pull request by stuartshields**

### CODE REVIEW

**Comment by stuartshields** [comment]
@BronsonQuick This now supports PHP 7.

File: modules/imagick/manifests/init.pp [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
-  package { 'php5-imagick':
-    ensure  => latest,
-    require => Package['php5-fpm'],
-    notify  => Service['php5-fpm']
-  }
+	if versioncmp( "${$php}", '5.4') <= 0 {
+		$php_package = 'php5'
+	}
+	else {
+		$php_package = "$php"
```
Comment (BronsonQuick) [inline modules/imagick/manifests/init.pp]: Can you please change this from `"$php"` to `"php$php"`

File: modules/imagick/manifests/init.pp [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+	}
+
+	package { "${$php_package}-imagick":
+		ensure  => latest,
+		require => Package["php${$php_package}-fpm"],
```
Comment (BronsonQuick) [inline modules/imagick/manifests/init.pp]: Can you change this and the line below to be `Package["${$php_package}-fpm"]` and I think we should be all good. Thanks!

**Review by BronsonQuick — CHANGES_REQUESTED** [review]

**Review by BronsonQuick — APPROVED** [review]

**Comment by BronsonQuick** [comment]
@stuartshields Thanks dude!

**Pull request merged**